snapshot/windows/windows.go

```diff
@@ -4,16 +4,33 @@ package windows
 
 import (
 	"context"
+	"os"
+	"path/filepath"
+	"strings"
+	"syscall"
+	"unsafe"
 
+	"golang.org/x/sys/windows"
+
+	"github.com/Microsoft/hcsshim"
+	"github.com/containerd/containerd/errdefs"
+	"github.com/containerd/containerd/fs"
+	"github.com/containerd/containerd/log"
 	"github.com/containerd/containerd/mount"
 	"github.com/containerd/containerd/plugin"
 	"github.com/containerd/containerd/snapshot"
+	"github.com/containerd/containerd/snapshot/storage"
 	"github.com/pkg/errors"
 )
 
-var (
-	ErrNotImplemented = errors.New("not implemented")
-)
+type snapshotter struct {
```

Comment:
Could you leave the type definition near the `New` function?

Reply:
will do